config-linux.md

```diff
@@ -645,6 +645,7 @@ The following parameters can be specified for the container:
     The value SHOULD start with `L3:` and SHOULD NOT contain newlines.
 * **`memBwSchema`** *(string, OPTIONAL)* - specifies the schema of memory bandwidth per L3 cache id.
     The value MUST start with `MB:` and MUST NOT contain newlines.
+* **`schemata`** *(array of strings, OPTIONAL)* - specifies the schemata to be written to the `schemata` file in resctrlfs. Each element represents one line in the `schemata` file. The value MUST NOT contain newlines.
```

Comment:
https://github.com/opencontainers/runtime-spec/blob/main/features-linux.md needs to cover the availability of this

Reply:
@AkihiroSuda PTAL